lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -4772,64 +4772,45 @@ namespace Js
         return returnValue;
     }
 
+    template<typename T, typename T(*func)(Var, ScriptContext*)> bool MemsetConversion(Var value, ScriptContext* scriptContext, T* result)
```

Comment:
clang is complaining about this line, it doesn't seem to like the `typename T(*func)(Var, ScriptContext*)`

```
[1m/mnt/j/w/Microsoft_ChakraCore/release_1.7/shared_ubuntu_linux_debug_prtest/lib/Runtime/Language/JavascriptOperators.cpp:4775:35: [0m[0;1;31merror: [0m[1mexpected a qualified name after 'typename' [-Werror][0m
12:35:58     template<typename T, typename T(*func)(Var, ScriptContext*)> bool MemsetConversion(Var value, ScriptContext* scriptContext, T* result)
```